migration/sql-files/029-identities-foreign-key.sql

```diff
@@ -0,0 +1,2 @@
+-- Refactor Identities: add a foreign key constraint
+alter table identities add constraint identies_user_id_users_id_foreign foreign key (user_id) REFERENCES users (id);
```

Comment:
I know, this is all theoretical but why keep a reference to something that doesn't exist? IMHO all FKs should specify some behavior in case of deletion. Maybe those other FKs you mentioned need to have a proper FK as well.

On the other hand I also understand why we should keep WIs even when the creator has been hard-deleted (which cannot happen anyway). There's potentially too much attached to a WI so it shouldn't be deleted automatically unless we follow a strict DB model from the start which seems pointless if hard-deletes are not possible. 

Then what about `ON DELETE RESTRICT`. Would that make more sense?